src/components/Datepicker/Datepicker.jsx

```diff
@@ -40,6 +40,9 @@ const DatePicker = ({
 }) => {
   const { themeName } = useTheme();
   const yearArr = useMemo(() => getYearArray(years), [years]);
+  useEffect(() => {
```

Comment:
make the useEffect a named function with a meaningful name describing
what the useEffect does, i'd add a comment as well.

Reply:
Because inside the useEffect function has only 1 line of code. So I think would add comment is enough